exercises/gigasecond/test/gigasecond_test.dart

```diff
@@ -1,47 +1,48 @@
+import 'package:dart2_constant/core.dart' as polyfill;
 import 'package:test/test.dart';
 import 'package:gigasecond/gigasecond.dart';
 
 void main() {
-  group('Add one gigasecond to the input.', gigasecondTests);
+  group("Gigasecond", gigasecondTests);
 }
 
 void gigasecondTests() {
   test("date only specification of time", () {
-    final DateTime birthDate = new DateTime.utc(2011, DateTime.APRIL, 25);
+    final DateTime birthDate = new DateTime.utc(2011, polyfill.DateTime.april, 25);
```

Comment:
If we're using the Dart 2 SDK, do we really need to use these polyfills? Or is it because you specified `>=1.24.0` in the version?

Reply:
Because we are still supporting Dart 1.0, then we need to use the polyfill (aka dart2_constant library).

Reply:
Ah, gotcha.